hydra/_internal/defaults_list.py

```diff
@@ -412,6 +433,16 @@ def _update_overrides(
                 overrides.add_override(parent.get_config_path(), d)
 
 
+def _has_config_content(cfg: DictConfig) -> bool:
+    if cfg is None or cfg._is_none():
```

Comment:
```suggestion
    if cfg._is_none():
```

Reply:
Sure. I added cfg._is_missing() to the check just in case someone adds a DictConfig("???") to the ConfigStore.